documentation/release_notes.markdown

```diff
@@ -16,6 +16,22 @@ canonical: "/puppetdb/latest/release_notes.html"
 [stockpile]: https://github.com/puppetlabs/stockpile
 [queue_support_guide]: ./pdb_support_guide.html#message-queue
 
+5.1.2
+-----
+
+PuppetDB 5.1.2 is a bugfix and performance release.
```

Comment:
We might well have used this construction before, but might be better to rewrite "performance release" somehow.

Reply:
As one possible example "PuppetDB 5.1.2 includes bugfixes and performance improvements".